docs/en_US/Compressor/Benchmark.md

```diff
@@ -0,0 +1,88 @@
+To provide an initial insight into the performance of various channel pruning algorithms, 
+we conduct extensive experiments with various pruning algorithms on some benchmark models and datasets.
+We present the experiment result in this document.
+In addition, we provide friendly instructions on the re-implementation of these experiments to facilitate further contributions to this effort.
+
+## Experiment Setting
+
+The experiments are performed with the following pruners/datasets/models:
+
+* Models: [VGG16, ResNet18, ResNet50](https://github.com/microsoft/nni/tree/master/examples/model_compress/models/cifar10)
+
+* Datasets: CIFAR-10
+
+* Pruners: 
+    - These pruners are included:
+        - Pruners with scheduling : SimulatedAnnealing, NetAdapt, AutoCompress
+        - One-shot pruners: L1Filter, L2Filter, FPGMPruner
```

Comment:
better to mention how to set each layer's sparsity for these one-shot pruners

Reply:
> better to mention how to set each layer's sparsity for these one-shot pruners

Added here.